getting-started/set-up-services.hbs.md

```diff
@@ -37,13 +37,14 @@ Before following this walkthrough, you must:
 1. Have access to a cluster with Tanzu Application Platform installed.
 1. Have downloaded and installed the Tanzu CLI and the corresponding plug-ins.
 1. Ensure that your Tanzu Application Platform cluster can pull the container images required by the Kubernetes operator providing the service. For more information, see:
-   * RabbitMQ Cluster Kubernetes operator in the [RabbitMQ documentation](https://www.rabbitmq.com/kubernetes/operator/using-operator.html).
+   * [VMware Tanzu RabbitMQ for Kubernetes](https://docs.vmware.com/en/VMware-Tanzu-RabbitMQ-for-Kubernetes/index.html).
    * [VMware Tanzu SQL with Postgres for Kubernetes](https://docs.vmware.com/en/VMware-Tanzu-SQL-with-Postgres-for-Kubernetes/index.html).
    * [VMware Tanzu SQL with MySQL for Kubernetes](https://docs.vmware.com/en/VMware-Tanzu-SQL-with-MySQL-for-Kubernetes/index.html).
 
 ## <a id="stk-set-up"></a> Set up a service
```

Comment:
@HenryBorys did you edit these changes to the notes?

Reply:
I went back in GitHub Blame and found that Ed King made those changes.

Reply:
I am sure he is a much better engineer than he is a writer. I'll fix this section tomorrow.